toolchains/gcc_compatibles.cmake

```diff
@@ -9,9 +9,10 @@
 # https://cmake.org/Bug/view.php?id=15939
 # http://stackoverflow.com/questions/31355692/cmake-support-for-gccs-link-time-optimization-lto
 set( CMAKE_INTERPROCEDURAL_OPTIMIZATION true )
-set( CMAKE_VISIBILITY_INLINES_HIDDEN    true )
+#set( CMAKE_VISIBILITY_INLINES_HIDDEN    true )
```

Comment:
Protobuf has problems when this is enabled - macOS linker throws lots of warnings.